chromedp.go

```diff
@@ -409,6 +411,71 @@ func WithBrowserOption(opts ...BrowserOption) ContextOption {
 	}
 }
 
+// RunResponse is an alternative to Run which can be used with a list of actions
+// that trigger a page navigation, such as clicking on a link or button.
+//
+// RunResponse will run the actions and block until a page loads, returning the
+// HTTP response information for its HTML document. This can be useful to wait
```

Comment:
not sure if the `ResourceTypeDocument` is only valid for HTML, how about any other document markup?

Reply:
Hmm. I tried to think about this, but since we want to wait for the DOM document of the main frame, can that really be anything other than a Document?

I could add a test for loading a txt file, though, to see what happens there.